### PR TITLE
Update ExperimentTabs to Use MUI Tabs

### PR DESCRIPTION
**Pull request by dhurlburtusa**
Update ExperimentTabs to Use MUI Tabs.

## How has this been tested?

Updated tests for component updates.
Manually tested.
Automated tests continue to pass with 100% coverage.

### CODE REVIEW

File: components/ExperimentTabs.test.tsx [hunk 1]
```diff
@@ -5,12 +5,6 @@ import { ExperimentFull, Platform, Status, Variation } from '@/models'
 
 import ExperimentTabs from './ExperimentTabs'
 
-test('renders nothing when experiment prop is `null`', () => {
```
Comment (dhurlburtusa) [inline components/ExperimentTabs.test.tsx:8]: No longer allow a `null` experiment prop.

File: components/ExperimentTabs.test.tsx [hunk 2]
```diff
@@ -42,8 +36,9 @@ test('renders expected links', () => {
     metricAssignments: [],
     segmentAssignments: [],
   })
-  const { getByText } = render(<ExperimentTabs experiment={experiment} />)
+  const { getByText } = render(<ExperimentTabs experiment={experiment} tab='details' />)
```
Comment (dhurlburtusa) [inline components/ExperimentTabs.test.tsx:39]: Currently, the tests don't depend on the various values for `tab` prop.

File: components/ExperimentTabs.tsx [hunk 3]
```diff
@@ -1,25 +1,65 @@
...
+import Tabs from '@material-ui/core/Tabs'
+import { useRouter } from 'next/router'
+import React, { ReactNode } from 'react'
 
 import { ExperimentFull } from '@/models'
 
+const useStyles = makeStyles(() =>
+  createStyles({
+    tab: {
+      minWidth: 110,
```
Comment (dhurlburtusa) [inline components/ExperimentTabs.tsx:12]: The default min-width was too big and didn't look right. This is an arbitrary size to looks good.

File: components/ExperimentTabs.tsx [hunk 4]
```diff
@@ -1,25 +1,65 @@
...
+    },
+  }),
+)
+
+function LinkTab({ as, label, url, value }: { as?: string; label: ReactNode; url: string; value: string }) {
+  const classes = useStyles()
+  const router = useRouter()
+  return (
+    <Tab
```
Comment (dhurlburtusa) [inline components/ExperimentTabs.tsx:21]: MUI `Tab` and the Next.js `Link` don't work well together. So, needed a special component to handle the routing and to render as MUI intended.

File: styles/theme.ts [hunk 5]
```diff
@@ -74,6 +74,9 @@ const theme = createMuiTheme({
     primary: {
       main: '#194661',
     },
+    secondary: {
```
Comment (dhurlburtusa) [inline styles/theme.ts:77]: The "current tab"-indicator uses the theme's secondary color. So, used a color to match the mockup. At this point, I am not sure what other MUI components use this color. So, we may need to make some adjustments if this unexpectedly change's some other MUI component's color.

**Comment by jessie-ross** [comment]
Looks fine for now.

**Pull request merged**